.github/workflows/comprehensive.yml

```diff
@@ -142,17 +142,15 @@ jobs:
         sudo rm -rf /usr/local/bin/node* || true &> /dev/null
 
     - name: INIT - install Node JS
-      run: |
-        curl -sL https://deb.nodesource.com/setup_18.x | sudo bash -
-        sudo apt-get install -y nodejs
-        sudo ln -s /usr/bin/node /usr/local/bin/node || true
-        
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18
+
     - name: INIT - install Node utilities
       run: |
-        sudo npm install --global npm
-        sudo npm install --global color-support
-        sudo npm install --global yarn
-        sudo npm install --global node-gyp
+        npm install --global npx || true
+        npm install --global yarn
```

Comment:
`yarn` should be included in setup-node 

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data

Reply:
Unfortunately, it's not included